go/cmd/vtctldclient/command/serving_graph.go

```diff
@@ -140,6 +148,35 @@ func commandGetSrvKeyspaces(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+var updateThrottlerConfigOptions = struct {
+	Enable            bool
+	Disable           bool
+	Threshold         float64
+	CustomQuery       string
+	CheckAsCheckSelf  bool
+	CheckAsCheckShard bool
+}{}
```

Comment:
any reason you're defining this as its own type vs

```go
var updateThrottlerConfigOptions vtctldatapb.UpdateThrottlerconfigRequest
```

?

(besides, "i just copied from another command" 😂 )

Reply:
I just copied from another command. Tell me what to do :joy:

Reply:
this is now cleaned up